includes/admin/functions.php

```diff
@@ -194,3 +194,22 @@ function amp_bootstrap_admin() {
 	$site_health = new SiteHealth();
 	$site_health->init();
 }
+
+/**
+ * Whether to activate the new onboarding feature.
+ *
+ * @todo Remove this function on release.
+ *
+ * @return boolean
```

Comment:
WPCS requires `bool` instead of `boolean`.
```suggestion
 * @return bool
```